doc/xml/daps_user_output.xml

```diff
@@ -235,4 +235,40 @@
    
   </variablelist>
  </sect1>
+ <sect1 id="sec.daps.user.output.xmlformat">
+  <title>Pretty XML Formatting</title>
+  <para>
+	  Sometimes it might be usefull to have propper formatted XML sources.
+	  It facilisate collaboration work and the usage of tools like <command>diff</command> or <command>git</command>.
```

Comment:
typo: "facilisate" -> "facilitates"
style: "collaboration work" -> either 1. "collaboration" or 2. "collaborative work" (i strongly vote for 1., because the "work" is sort of implied)

A reasoning would also be interesting, such as "makes diffs more readable" and "reduces the amount of changes that need to be stored in version control systems which leads to smaller reppositories, if done consistently"